python/src/RandomWalkMetropolisHastings_doc.i.in

```diff
@@ -51,163 +32,162 @@ proposal : list of :class:`~openturns.Distribution`
     :math:`\delta_j^k` of :math:`\vect{\delta}^k` , with :math:`j=k \mod d`, is
     not zero and :math:`\delta_j^k = \lambda_j^k \delta^k` where
     :math:`\lambda_j^k` is a deterministic scalar *calibration* coefficient and
-    where :math:`\delta^k \sim p_j`. Moreover, :math:`\lambda_j^k = 1` by default,
-    but adaptive strategy based on the acceptance rate of each component can be
-    defined using the method :meth:`setCalibrationStrategyPerComponent`.
+    where :math:`\delta^k \sim p_j`. Moreover, :math:`\lambda_j^k = 1` by default.
+marginalIndices : sequence of int, optional
+    Bloc indices to indicate which components are updated
+    If not provided this means all components
 
 Notes
 -----
-A RandomWalkMetropolisHastings enables to carry out :class:`~openturns.MCMC`
```

Comment:
Again, one new realization now comes down to carrying out a single Metropolis-Hastings iteration, instead of d